src/post_processing.jl

```diff
@@ -478,31 +340,98 @@ function _reactive_power_redistribution_pv(
     return
 end
 
+function _set_branch_flow!(br::PSY.Branch, flow::Complex)
+    PSY.set_active_power_flow!(br, real(flow))
+    PSY.set_reactive_power_flow!(br, imag(flow))
+    return
+end
+
+function _set_branch_flow!(tp::Tuple{PSY.ThreeWindingTransformer, Int}, flow::Complex)
+    (trf, winding) = tp
+    if winding == 1
+        PSY.set_active_power_flow_primary!(trf, real(flow))
+        PSY.set_reactive_power_flow_primary!(trf, imag(flow))
+    elseif winding == 2
+        PSY.set_active_power_flow_secondary!(trf, real(flow))
+        PSY.set_reactive_power_flow_secondary!(trf, imag(flow))
+    elseif winding == 3
+        PSY.set_active_power_flow_tertiary!(trf, real(flow))
+        PSY.set_reactive_power_flow_tertiary!(trf, imag(flow))
+    else
+        error("Invalid winding number: $winding")
+    end
+    return
+end
+
+function set_branch_flows_for_dict!(
+    d::Union{
```

Comment:
This is a weird union that likely means we need to do something smarter. Union of 2 is ok, 3 is already at the limit and this combo has too many abstracts on it.